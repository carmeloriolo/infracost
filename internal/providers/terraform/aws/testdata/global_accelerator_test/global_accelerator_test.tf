provider "aws" {
  region                      = "us-east-1"
  skip_credentials_validation = true
  skip_metadata_api_check     = true
  skip_requesting_account_id  = true
  skip_get_ec2_platforms      = true
  skip_region_validation      = true
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
}

# Add example resources for GlobalAccelerator below

resource "aws_globalaccelerator_accelerator" "example" {
  name            = "Example"
  ip_address_type = "IPV4"
  enabled         = true

  #attributes {
    #flow_logs_enabled   = true
    #flow_logs_s3_bucket = "example-bucket"
    #flow_logs_s3_prefix = "flow-logs/"
  #}
}
