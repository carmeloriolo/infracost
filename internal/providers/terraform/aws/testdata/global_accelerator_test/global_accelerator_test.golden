
 Name          Monthly Qty  Unit   Monthly Cost 
                                                
 Example                                        
 └─ Fixed Fee          730  hours        $18.25 
                                                
 OVERALL TOTAL                           $18.25 
──────────────────────────────────
1 cloud resource was detected:
∙ 1 was estimated